forward_engineering/ddlProvider/ddlProvider.js

```diff
@@ -490,7 +490,7 @@ module.exports = (baseProvider, options, app) => {
 		},
 
 		createIndex(tableName, index, dbData, isParentActivated = true) {
-			const name = getIndexName(index.indxName);
+			const name = getNamePrefixedWithSchemaName(index.indxName, index.schemaName);
```

Comment:
seems like there might be a problem here. In the original `getIndexName` if there was no index name - it returns empty string. Where with the new approach of reusing `getNamePrefixedWithSchemaName`, it is possible that in case of empty `index.indxName` **and** non-empty `index.schemaName` - it will return something like:
```
"a_schema".""
```